### DISCUSSION TITLE
Feature request: replace button with text after date has passed

### DISCUSSION THREAD

**Discussion by apos37** [add2cal/add-to-calendar-button#483] [Ideas / Requests]
I see that you have a `pastDateHandling` option that allows us to disable or hide the button when the date/time has passed, but I would love to replace it with a custom message. For instance, once our event has passed, we add a message saying, "The live zoom meeting is now over. A recorded video will be shown here when it is ready." Just a new option for a custom text would suffice, maybe wrapped in a `div` with a class so it can be styled. Support for html, links, and shortcodes inside that text would be even better.

**Comment by jekuer** [comment]
There would be a workaround to achieve this. You can make use of the customLabels option and override the "expired" label.
See https://add-to-calendar-button.com/configuration#customlabels
